docs/building/README.md

```diff
@@ -0,0 +1,24 @@
+# Building
+
+> Part of Open Web Component Recommendation [open-wc](https://github.com/open-wc/open-wc/)
```

Comment:
this header is only needed for readmes of individual packages to show them in npmjs.org
here we can remove them